rend3/src/graph/graph.rs

```diff
@@ -339,12 +339,13 @@ impl<'node> RenderGraph<'node> {
             match *sub_resource {
                 GraphSubResource::Texture(region) => {
                     if let Entry::Vacant(vacant) = active_views.entry(region) {
-                        let view = active_textures[&region.idx].create_view(&TextureViewDescriptor {
+                        let texture = &active_textures[&region.idx];
+                        let view = texture.create_view(&TextureViewDescriptor {
                             base_array_layer: region.layer_start,
                             array_layer_count: Some(NonZeroU32::new(region.layer_end - region.layer_start).unwrap()),
                             ..TextureViewDescriptor::default()
                         });
-                        vacant.insert(view);
+                        vacant.insert((view, Arc::clone(texture)));
```

Comment:
I tried to make the change as unintrusive as possible, so now the `active_views` stores both a `TextureView` and an `Arc<Texture>` with the texture it was created from as a tuple.